_includes/topnav.html

```diff
@@ -0,0 +1,38 @@
+<nav class="navbar navbar-default navbar-fixed-top">
+    <div class="container navbar-container">
+        <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+            <a class="navbar-brand" href="{{ site.baseurl }}/">
+                <span><img src="{{site.baseurl}}/img/logonav.png"></span> {{ site.title }}
+            </a>
+        </div>
+        <div id="navbar" class="collapse navbar-collapse">
+            <ul class="nav navbar-nav">
+                <li><a href="{{ "/docs/home/" | prepend: site.baseurl }}">Get started</a></li>
+                <li><a href="{{ "/docs/example-payments/" | prepend: site.baseurl }}">Examples</a></li>
+                <li><a href="{{ "/docs/sdks/" | prepend: site.baseurl }}">SDKs</a></li>
+                <li><a href="{{ "/docs/architecture/" | prepend: site.baseurl }}">Architecture</a></li>
+                <li><a href="{{ "/docs/payout-details/" | prepend: site.baseurl }}">Payments</a></li>
+                <li><a href="{{ "/docs/collection-details/" | prepend: site.baseurl }}">Collections</a></li>
+                <li><a href="{{ "/docs/reference-documentation/" | prepend: site.baseurl }}">Reference</a></li>
+            </ul>
+            <div class="navbar-right">
+                <form class="navbar-form navbar-left" method="get" action="http://www.google.com/search" target="_blank">
```

Comment:
@sztupy I noticed I couldn't search yet. Will we add the site in google search console / webmaster? I think there's a file we need to add to the root of the page.

Reply:
Yeah, I wanted to do that once this is approved. We might add a CNAME as well so this actually moves to `transferzero.com`